tests/certification/state/azure/tablestorage/tablestorage_test.go

```diff
@@ -0,0 +1,178 @@
+
+		return nil
+	}
+
+	NonExistingTableTest := func(ctx flow.Context) error {
+		client, err := client.NewClientWithPort(fmt.Sprint(currentGrpcPort))
+		if err != nil {
+			panic(err)
+		}
+		defer client.Close()
+
+		// save state, default options: strong, last-write
+		err = client.SaveState(ctx, "statestore-newtable", certificationTestPrefix+"key1", []byte("certificationdata"), nil)
+		assert.NoError(t, err)
+
+		// get state
+		item, err := client.GetState(ctx, "statestore-newtable", certificationTestPrefix+"key1", nil)
+		assert.NoError(t, err)
+		assert.Equal(t, "certificationdata", string(item.Value))
+
+		// delete state
+		err = client.DeleteState(ctx, "statestore-newtable", certificationTestPrefix+"key1", nil)
+		assert.NoError(t, err)
+
+		exec.Command("/bin/bash", "az storage table delete --account-name dapr3ctstorage --name NewTable")
+
+		return nil
+	}
+
+	authTest := func(ctx flow.Context) error {
+		client, err := client.NewClientWithPort(fmt.Sprint(currentGrpcPort))
+		if err != nil {
+			panic(err)
+		}
+		defer client.Close()
+
+		// save state with go sdk with existing table
+		err = client.SaveState(ctx, "statestore-key", certificationTestPrefix+"key1", []byte("certificationdata"), nil)
+		assert.NoError(t, err)
+
+		// Get data using AAD
+		item, err := client.GetState(ctx, "statestore-aad", certificationTestPrefix+"key1", nil)
+		assert.NoError(t, err)
+		assert.Equal(t, "certificationdata", string(item.Value))
+
+		return nil
+	}
+
+	flow.New(t, "Test basic operations, save/get/delete using existing table").
+		// Run the Dapr sidecar with azure table storage.
+		Step(sidecar.Run(sidecarNamePrefix,
+			embedded.WithoutApp(),
+			embedded.WithDaprGRPCPort(currentGrpcPort),
+			embedded.WithDaprHTTPPort(currentHTTPPort),
+			embedded.WithComponentsPath("./components/basictest"),
+			runtime.WithSecretStores(
+				secretstores_loader.New("local.env", func() secretstores.SecretStore {
+					return secretstore_env.NewEnvSecretStore(log)
+				}),
+			),
+			runtime.WithStates(
+				state_loader.New("azure.tablestorage", func() state.Store {
+					return table.NewAzureTablesStateStore(log)
+				}),
+			))).
+		Step("Run basic test with existing table", basicTest).
+		Run()
+
+	flow.New(t, "Test basic operations, save/get/delete with new table").
+		// Run the Dapr sidecar with azure table storage.
+		Step(sidecar.Run(sidecarNamePrefix,
+			embedded.WithoutApp(),
+			embedded.WithDaprGRPCPort(currentGrpcPort),
+			embedded.WithDaprHTTPPort(currentHTTPPort),
+			embedded.WithComponentsPath("./components/nonexistingtabletest"),
+			runtime.WithSecretStores(
+				secretstores_loader.New("local.env", func() secretstores.SecretStore {
+					return secretstore_env.NewEnvSecretStore(log)
+				}),
+			),
+			runtime.WithStates(
+				state_loader.New("azure.tablestorage", func() state.Store {
+					return table.NewAzureTablesStateStore(log)
+				}),
+			))).
+		Step("Run basic test with new table", NonExistingTableTest).
```

Comment:
This is where you can call the test function with the parameter, and then add a second step for the deletion